test/test_cabin_helper.rb

```diff
@@ -0,0 +1,199 @@
+$: << File.dirname(__FILE__)
+$: << File.join(File.dirname(__FILE__), "..", "lib")
+
+require "rubygems"
+require "minitest-patch"
+require "cabin"
+
+module RSpec
+  class Confi
+    def include(*)
```

Comment:
not sure I understand why do you need to void all this methods in here? can you educate me please?